asdf/tags/core/tests/test_history.py

```diff
@@ -69,7 +69,7 @@ def test_history_to_file(tmpdir):
     with asdf.AsdfFile() as ff:
         ff.add_history_entry('This happened',
                              {'name': 'my_tool',
-                              'homepage': 'http://nowhere.com',
+                              'homepage': 'http://nowhere.org',
```

Comment:
This only works until they also buy nowhere.org... :thinking:

Reply:
This isn't validated anywhere. The only one that actually matters for testing purposes is the one in the diff below.